pkg/apis/pipeline/v1alpha1/pipelinerun_types_test.go

```diff
@@ -130,6 +160,110 @@ func TestPipelineRunIsCancelled(t *testing.T) {
 	}
 }
 
+func TestPipelineRunIsFailed(t *testing.T) {
+	params := []struct {
+		name          string
+		prStatus      v1alpha1.PipelineRunStatus
+		expectedValue bool
+	}{
+		{
+			"prWithNoStatus",
```

Comment:
Can we add field names to these? `name: "prWithNoStatus",`,etc.

These make it easier to read, and make it easier to add a fourth field in the future.